app/experimenter/targeting/constants.py

```diff
@@ -113,6 +113,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_REDUCED_MOTION = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default, preferring reduced motion"
+    ),
+    slug="first_run_new_profile_need_default_prefers_reduced_motion",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default, preferring reduced motion"
+    ),
+    targeting=(
+        "{first_run} && os.windowsBuildNumber >= 18362 && {new_profile} && "
+        "{need_default} && !userPrefersReducedMotion".format(
```

Comment:
Can we have  FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting  similar to FIRSTRUN_targeting on Line 102 to replace `{first_run} && os.windowsBuildNumber >= 18362 && {new_profile} && "
        "{need_default} `